rad/prelude/chain.rad

```diff
@@ -44,9 +44,12 @@
     {:chain new-chain
      :result result }))
 
+(def local-test-chain-name
+  (fn [] (string-append "http://localhost:8024/chains/" (uuid!))))
```

Comment:
Isn't the default port the `server` runs on `8000`?

Reply:
I’m using the port that is exposed by `docker-compose.yaml`.

Reply:
Can we make that consistent with the default port? Otherwise there's an extra bit of information that needs to be conveyed for running the tests locally.